integration/simulation/simulation.go

```diff
@@ -65,23 +65,21 @@ func (s *Simulation) Start() {
 	s.prefundL1Accounts()      // Prefund every L1 wallet
 
 	timer := time.Now()
-	fmt.Printf("Starting injection\n")
```

Comment:
This (and the other prints) was useful to see from the console that the sim was progressing.

Reply:
added back